src/tests/ftest/object/ObjOpenBadParam.py

```diff
@@ -27,52 +27,23 @@
 import sys
 import json
 import logging
-from avocado import Test, main, skip
-
-sys.path.append('./util')
-sys.path.append('../util')
-sys.path.append('../../../utils/py')
-sys.path.append('./../../utils/py')
+from apricot import TestWithServers
+from avocado import main
```

Comment:
(pylint-import-error) Unable to import 'avocado'

(pylint-unused-import) Unused main imported from avocado